naga-cli/src/bin/naga.rs

```diff
@@ -529,6 +529,7 @@ fn write_output(
                     pipeline_options_owned = spv::PipelineOptions {
                         entry_point: name.clone(),
                         shader_stage: module.entry_points[ep_index].stage,
+                        constants: naga::back::PipelineConstants::default(),
```

Comment:
question: Why not use `Default::default()` like in other parts of the commit? 🤔

This question also applies to changes in `naga/benche/criterion.rs`.